json-schemas/src/client-events-api-requests.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Ably API Requests Client Event",
+  "type": "object",
+  "description": "A message sent on the clientEvents:apiRequests metachannel describing a sampled client api request.",
+  "properties": {
+    "host": {
+      "type": "string",
+      "description": "The HTTP Host header of the API request."
+    },
+    "requestId": {
+      "type": "string",
+      "description": "The unique ID of the request that Ably can use to correlate a request event with internal logs, if necessary."
+    },
+    "site": {
+      "type": "string",
+      "description": "The site the request was processed in."
+    },
+    "headers": {
+      "type": "object",
+      "description": "The HTTP headers sent with the request. Keys should be lower case."
+    },
+    "query": {
+      "type": "object",
+      "description": "The details of the parsed request query, including information such as the request format."
+    },
+    "path": {
+      "type": "string",
+      "description": "The path of the endpoint called in the request, for example /channels/{channel-name}/messages."
+    },
+    "channels": {
+      "type": "array",
+      "description": "A list of channels the request was made against."
```

Comment:
I don't like the term "made against", how about "references"?

"The list of channels the API request references."